options.go

```diff
@@ -85,16 +92,47 @@ type Options struct {
 	// Has no effect for single connections.
 	Concurrency uint
 
-	// Read the TLS certificate from this file
+	// Parameters used to configure TLS connections to EdgeDB server.
+	TLSOptions TLSOptions
```

Comment:
This doesn't seem to allow passing your own `tls.Config`. Maybe this could be an interface with one method `TLSConfig() (tls.Config, error)` or similar and then add that method to the TLSOptions type? Is there another plan for accepting user defined  `tls.Config`?

Reply:
`TLSOptions` can embed `tls.Config` in the future.  This PR doesn't deal with that just yet.